src/components/plots/styling/violin/ViolinControls.jsx

```diff
@@ -5,45 +5,42 @@ import {
 } from 'antd';
 import SelectData from 'components/plots/styling/SelectData';
 import MultiViewEditor from 'components/plots/styling/MultiViewEditor';
+import { plotUuids, plotTypes } from 'utils/constants';
 import GeneSearchBar from 'components/plots/GeneSearchBar';
 
 const { Panel } = Collapse;
-
+const plotUuid = plotUuids.VIOLIN_PLOT;
+const plotType = plotTypes.VIOLIN_PLOT;
 const ViolinControls = (props) => {
   const {
+    experimentId,
     config,
-    multiViewConfig,
-    onUpdate,
     onUpdateConditional,
     updateAll,
     setUpdateAll,
-    onMultiViewUpdate,
     selectedPlotUuid,
     setSelectedPlotUuid,
-    addGeneToMultiView,
     cellSets,
     shownGenes,
+    changeFirstPlotGene,
```

Comment:
Before with the multiview it was possible to select which plot's gene to change I believe, is that behavior preserved?

Reply:
I just checked and it is, I think the name `changeSelectedPlotGene` makes more sense